src/config.h

```diff
@@ -57,6 +57,11 @@ typedef enum {
 	GRANT_MANUAL,
 } grant_type_e;
 
+typedef enum {
+	TICKET_MODE_AUTO = 1,
```

Comment:
Doesn't pratically matter, but 0 seems good for the first enumerator to me ;-)

Reply:
Well, you would be surprised ;) 

> [...]
> typedef enum {
> 	ATTR_OP_EQ = 1,
> 	ATTR_OP_NE,
> } attr_op_e;
> 
> typedef enum {
> 	GRANT_AUTO = 1,
> 	GRANT_MANUAL,
> } grant_type_e;
> 
> typedef enum {
> 	TICKET_MODE_AUTO = 1,
> 	TICKET_MODE_MANUAL,
> } ticket_mode_e;
> [...]